lib/Core/Executor.cpp

```diff
@@ -1249,40 +1250,46 @@ void Executor::executeCall(ExecutionState &state,
           // Alignment requirements for scalar types is the same as their size
           if (argWidth > Expr::Int64) {
              size = llvm::RoundUpToAlignment(size, 16);
```

Comment:
It's the AMD64 calling convention. I didn't know either.
On Apr 19, 2016 7:41 PM, "Dan Liew" <notifications@github.com> wrote:

> In lib/Core/Executor.cpp
> <https://github.com/klee/klee/pull/363#discussion_r60276354>:
>
> > @@ -1249,40 +1250,46 @@ void Executor::executeCall(ExecutionState &state,
> >            // Alignment requirements for scalar types is the same as their size
> >            if (argWidth > Expr::Int64) {
> >               size = llvm::RoundUpToAlignment(size, 16);
>
> Why 16?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/klee/klee/pull/363/files/864eebf0da3f9edb756a55ff8baacc7deb02b931#r60276354>
>